05_stringer/shuutech/main_test.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"testing"
+)
+
+func TestIpAddress(t *testing.T) {
+	var expected string
+	var actual string
+	expected = "127.0.0.1"
+	actual = IPAddr{127, 0, 0, 1}.String()
+	if expected != actual {
+		t.Errorf("expected %v, got %v", actual, expected)
+		t.Fail()
+	}
+}
+
+func TestMain(m *testing.M) {
```

Comment:
This test will never fail. 

You can make main function testable using following printf method.
`fmt.Fprintln(out, IPAddr{127, 0, 0, 1})`